aggregator/src/aggregator/aggregation_job_writer.rs

```diff
@@ -507,7 +507,7 @@ where
                             aggregation_job: Cow::Borrowed(aggregation_job),
                             report_aggregations: report_aggregations
                                 .iter()
-                                .map(Cow::Borrowed)
+                                .map(RA::borrow)
```

Comment:
Check my understanding: at this point, report_aggregations is (roughly) a `Vec<impl ReportAggregationUpdate>`, which, concretely, could be a `struct ReportAggregationMetadata` or a `Cow` of it?

In which case a naive `map(Cow::Borrowed)` would double `Cow` a `Cow<'_, ReportAggregationMetadata>`, so we need a trait method to avoid this.

Reply:
That's correct. The implementation of `ReportAggregationUpdate` over `Cow` exists to allow a `Cow<WritableReportAggregation>` (or a `Cow` of anything else that implements `ReportAggregationUpdate`), _without_ needing to deal with `Cow<Cow<T>>`.